apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/CustomEnumTypeSpecBuilder.kt

```diff
@@ -37,10 +37,15 @@ class CustomEnumTypeSpecBuilder(
               .addStatement("return \$S", scalarType)
               .build())
           .addMethod(MethodSpec.methodBuilder("javaType")
-              .addModifiers(Modifier.PUBLIC)
-              .addAnnotation(Override::class.java)
-              .returns(Class::class.java)
-              .addStatement("return \$T.class", javaTypeName.toJavaType())
+                  .addModifiers(Modifier.PUBLIC)
+                  .apply {
+                    if (context.suppressRawTypesWarning) {
+                      this.addAnnotation(com.apollographql.apollo.compiler.Annotations.SUPPRESS_WARNINGS)
+                    }
+                  }
+                .addAnnotation(Override::class.java)
+                .returns(Class::class.java)
+                .addStatement("return \$T.class", javaTypeName.toJavaType())
```

Comment:
The indentation seems to be off here as well - can you adjust it to match the rest of the file?